jonprl-mode.el

```diff
@@ -48,6 +48,23 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   ("<>" . ?★)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   ("base" . ?𝔻)
+   ("fix" . ?μ)
```

Comment:
Lastly, I am not sure that we want this one, since it might get confused with a facility for defining least fixed points of functors, which we don't currently have.